content/java/concepts/strings/terms/format/format.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: '.format()'
+Description: 'Returns a string with additional arguments in a specifically defined format.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.format()` method returns a string with additional arguments in a specifically defined format.
+
+## Syntax
+
+```pseudo
+string.format(String str, ...args)
+```
+
+- `str` is of type `String` and includes a **format specifier**.
+- `args` is an optional argument which can be of any data type to be returned with the first string argument.
+
+## Format Specifiers
```

Comment:
```suggestion
### Format Specifiers
```

Wondering if this should be a lv. 3 header since this sort of falls under "syntax"?